javatests/google/registry/rde/imports/XjcToDomainResourceConverterTest.java

```diff
@@ -120,6 +120,8 @@ public void testConvertDomainResource() throws Exception {
     assertThat(domain.getLastEppUpdateTime()).isNull();
     assertThat(domain.getAutorenewBillingEvent()).isNotNull();
     assertThat(domain.getAutorenewPollMessage()).isNotNull();
+    assertThat(domain.getAuthInfo()).isNotNull();
+    assertThat(domain.getAuthInfo().getPw().getValue()).hasLength(16);
```

Comment:
You can use DeterministicStringGenerator to make a stronger assertion here (that it is an exact value).  That is the intent behind said class; to be used for tests.